Framework/DataHandling/inc/MantidDataHandling/AlignAndFocusPowderSlim/ProcessBankSplitFullTimeTask.h

```diff
@@ -0,0 +1,66 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2025 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+
+#include "MantidAPI/MatrixWorkspace.h"
+#include "MantidDataHandling/AlignAndFocusPowderSlim/NexusLoader.h"
+#include "MantidDataHandling/DllConfig.h"
+#include "MantidGeometry/IDTypes.h"
+#include "MantidKernel/DateAndTime.h"
+#include <H5Cpp.h>
+#include <MantidAPI/Progress.h>
+#include <map>
+#include <set>
+#include <tbb/tbb.h>
+#include <vector>
+
+namespace Mantid::DataHandling::AlignAndFocusPowderSlim {
+
+// we need to include additional pulses because time-of-flight can be greater than a couple of pulse periods
+const int64_t PULSETIME_OFFSET = 66666666;
```

Comment:
This might be easier to understand if it was defined as 
```suggestion
constexpr int64_t PULSETIME_OFFSET{16667*400};
```
so people could more clearly see that it is arbitrarily big

Reply:
It's in nanoseconds so it's only 4 pulses long. I updated the comment to make that clear.